content/contribute/contribute_dns.md

```diff
@@ -0,0 +1,49 @@
+---
+title: DNS
+linktitle: DNS for contributors
+description: How to configure DNS for developing features, reproducing issues and demos
+date: 2018-12-18
+publishdate: 2018-12-18
+lastmod: 2018-12-18
+categories: [contribute_dns]
+keywords: [dns]
+authors: [pmuir]
+menu:
+  docs:
+    parent: "contribute"
+    weight: 30
+weight: 30
+sections_weight: 30
+draft: false
+toc: true
+---
+
+Some features and issues require DNS to be configured. The domain name `jenkins-x.rocks` is available for
+use.
+
+To add records, visit [the Cloud DNS dashboard](https://console.cloud.google.com/net-services/dns/zones/jenkins-x-rocks?project=jenkins-x-rocks&organizationId=41792434410). 
```

Comment:
FWIW we have a command to do this

```
jx create domain gke --domain jenkins-x.rocks
```
Am gonna try a live demo of this tomorrow at JW.

Reply:
This works with external-dns BTW

Reply:
Yeah, I should probably mention external dns too as people will want to debug that too. I'll ask Cai to help.

Does the `jx create domain` work even if you aren't connected to a cluster?

Reply:
@cagiti how can we improve this to discuss external dns? Goal here is a contributor guide for debugging so trying to cover all paths.